settings.py

```diff
@@ -0,0 +1,20 @@
+import os
+
+from dotenv import load_dotenv
+from pydantic.v1 import BaseSettings
+
+
+load_dotenv()
+
+
+class Settings(BaseSettings):
+    DATABASE_URL: str | None = "sqlite+aiosqlite:///./city_temperature.db"
+    API_PREFIX = "/api/v1"
+    WEATHER_TOKEN = os.getenv("WEATHER_TOKEN")
```

Comment:
The `WEATHER_TOKEN` is retrieved using `os.getenv`. Ensure that the `.env` file contains the `WEATHER_TOKEN` variable, or it will default to `None`.